lib/rubocop/cop/style/trailing_body_on_method_definition.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for trailing code after the method definition.
+      #
+      # @example
+      #   # bad
+      #   def some_method; do_stuff
+      #   end
+      #
+      #   # good
+      #   def some_method
+      #     do_stuff
+      #   end
+      #
+      class TrailingBodyOnMethodDefinition < Cop
+        include AutocorrectAlignment
+
+        MSG = 'Method body goes below definition.'.freeze
+
+        def on_def(node)
+          return unless node.body
+          return unless trailing_body?(node)
+
+          add_offense(node)
+        end
+        alias on_defs on_def
```

Comment:
I am unsure about the need for this alias. I see it on a lot of cops but am yet not sure what it is achieving.

Reply:
It applies the cop to class method definitions as well.